app/controllers/course/achievements_controller.rb

```diff
@@ -3,6 +3,7 @@ class Course::AchievementsController < Course::ComponentController
   add_breadcrumb :index, :course_achievements_path
 
   def index #:nodoc:
+    @achievements = @achievements.includes(:conditions)
```

Comment:
Is this not needed? It's for showing conditions per achievement [here](https://github.com/coursemology-collab/coursemology2/blob/dariusf/condition-refinements/app/views/course/achievements/_achievement.html.slim#L11).